services/spar/src/Spar/Data.hs

```diff
@@ -215,6 +217,13 @@ deleteUsersByIssuer issuer = retry x5 . write del $ params Quorum (Identity issu
     del :: PrepQuery W (Identity SAML.Issuer) ()
     del = "DELETE FROM user WHERE issuer = ?"
 
+-- | Delete a user from the saml users table.
+deleteSAMLUser :: (HasCallStack, MonadClient m) => SAML.UserRef -> m ()
```

Comment:
Sure; the reason I'm changing from the convention here is that the linter prevented me from adding another `deleteUser`; then I shied away from the big renaming because we'd agreed to not do refactoring along with features; but the separated diff idea sounds good :)

Reply:
I'm in favor of adding `SAML` to all names (in a separate PR).  Always a big fan of long names.  ;-)